src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -75,7 +75,21 @@
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
-        AssemblyName="$(AssemblyName)" />
+        AssemblyName="$(AssemblyName)" >
+
+        <Output TaskParameter="NormalizedAssemblyName" PropertyName="_NormalizedAssemblyName" />
```

Comment:
@weshaggard I'll not go with your idea now because we'll need to have a way to handle such cases when executing the resources extraction that @safern did as we'll not have the needed information (e.g. extracting the resources of the runtime assemblies on WCF repro). I'll still think about the idea to see if we can have a reliable way to do it.